.codespell/dictionary.txt

```diff
@@ -0,0 +1 @@
+openverse->Openverse
```

Comment:
Nice, I was also thinking we could add these (if not already) and document this file in the Sphinx docs so that we, and new contributors, know where to add more definitions in the future.

```
wordpress->WordPress
typescript->TypeScript
javascript->JavaScript
django->Django
```

Reply:
Unfortunately codespell does not match case on the left-hand side :cry: Hence the issue with the `afterAll` Jest hook getting confused with `afterall->after all`. This Openverse one doesn't even work anyway, or else there would be a lot of places to add ignores for.